docusaurus/src/css/custom.css

```diff
@@ -279,4 +279,8 @@ table tr:hover {
 }
 table thead tr:hover {
   background-color: transparent;
+}
+
+.markdown li > p {
```

Comment:
This is not related to anything, but it fixes an oddity where list items in the final docs had variable top and bottom margins depending on how an author spaced things out in their MarkDown.